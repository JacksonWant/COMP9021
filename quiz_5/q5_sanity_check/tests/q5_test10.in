13609683913728
