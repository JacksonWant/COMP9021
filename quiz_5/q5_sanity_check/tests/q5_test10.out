Input either a strictly positive integer
or a nonempty list of strictly positive integers:   In base 2, 13609683913728 reads as 11000110000011000000011000000000110000000000
  It encodes:  [2, 4, 8, 16, 32]
